src/components/App/index.tsx

```diff
@@ -38,4 +38,9 @@ const App = (props: AppProps): any => {
     </Provider>
   )}
 
+// history listen on change
+history.listen((loc: any) => {
+  loc.search.includes('retrox') && document.body.classList.add('RETROX')
```

Comment:
I though about it and a better, more compatible way would be
```js
const searchParams = new URLSearchParams(location.search)
document.body.classList.toggle('RETOX', searchParams.has('retox'))
```

Reply:
works well!